.github/workflows/embedded_ui_refresh.yaml

```diff
@@ -0,0 +1,71 @@
+on:
+  repository_dispatch:
+    types: [embedded_ui_refresh]
+  workflow_dispatch:
+    inputs:
+      asset_name:
+        description: Release asset name
+        required: true
+        default: embedded-ui
+      repository:
+        description: Source repository
+        required: true
+        default: ydb-platform/ydb-embedded-ui
+      tag_name:
+        description: Release tag
+        required: true
+
+name: Embedded UI Refresh
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    env:
+      ASSET_NAME: ${{ github.event.inputs && github.event.inputs.asset_name || github.event.client_payload.asset_name }}
+      REPOSITORY: ${{ github.event.inputs && github.event.inputs.repository || github.event.client_payload.repository }}
+      TAG_NAME: ${{ github.event.inputs && github.event.inputs.tag_name || github.event.client_payload.tag_name }}
+      TEMP_ASSET_DIR: /tmp/asset
+    steps:
+      - uses: actions/checkout@v4
+      - name: Asset Download
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: gh release download $TAG_NAME --repo $REPOSITORY --pattern $ASSET_NAME.zip --dir $TEMP_ASSET_DIR
+      - name: Asset Placement
+        env:
+          START_POINTER: "# GENERATED MONITORING RESOURCES START"
+          END_POINTER: "# GENERATED MONITORING RESOURCES END"
+          TARGET_DIR: ydb/core/viewer/monitoring
+          YA_MAKE_FILE: ydb/core/viewer/ya.make
+        run: |
+          unzip $TEMP_ASSET_DIR/$ASSET_NAME.zip -d $TEMP_ASSET_DIR
+          rm -rf $TARGET_DIR
+          mkdir $TARGET_DIR
+          mv -vf $TEMP_ASSET_DIR/$ASSET_NAME/* $TARGET_DIR
+
+          NEW_RESOURCES=$(find $TARGET_DIR -type f | grep -v '.map' | sort)
```

Comment:
In the initial instruction also `*. LICENSE *` files were excluded

Reply:
Decided that we need to keep these files in order to comply with all licenses.

Also I removed `.map` filter because there are no any source maps in the build.